packages/database/src/api/Query.ts

```diff
@@ -526,6 +529,28 @@ export class Query {
       value = null;
       name = null;
     }
+
+    return new Query(this.repo, this.path, newParams, this.orderByCalled_);
+  }
+
+  startAfter(
+    value: number | string | boolean | null = null,
+    name?: string | null
+  ): Query {
+    validateArgCount('Query.startAfter', 0, 2, arguments.length);
+    validateFirebaseDataArg('Query.startAfter', 1, value, this.path, true);
```

Comment:
This argument isn't optional (it can be null, but not undefined). Not sure why it is marked optional in startAt.